Makefile

```diff
@@ -1,27 +1,28 @@
-egg-info:
-	pip install -q -e .
+app:
+	gunicorn -w 5 -b 0.0.0.0:8000 --log-level info --log-file /var/log/gunicorn.log aleph.manage:app
 
-web: egg-info assets
-	python aleph/manage.py runserver -h 0.0.0.0 -p 8000
-
-worker: egg-info
+worker:
 	celery -A aleph.queues -B -c 4 -l INFO worker --pidfile /var/lib/celery.pid
 
-beat: egg-info
+beat:
 	celery -A aleph.queues beat -s /var/lib/celerybeat-schedule.db --pidfile /var/lib/celery.pid
 
-clear: egg-info
+clear:
 	celery purge -f -A aleph.queues
 
-assets:
+install:
+	pip install -r requirements.txt
+	pip install -e .
+	npm install
 	touch aleph/static/style/_custom.scss;
 	(test -f '$(CUSTOM_SCSS_PATH)' && cp -f $(CUSTOM_SCSS_PATH) aleph/static/style/_custom.scss) || exit 0
-	/node_modules/webpack/bin/webpack.js --env.prod
+	./node_modules/webpack/bin/webpack.js --env.prod
```

Comment:
This pull request is about partly about fixing that problem so we don't have to use a hack to install node_modules globally